### DISCUSSION TITLE
cache 失效

### DISCUSSION THREAD

**Discussion by AgileZM** [midwayjs/midway#3755] [Q&A]
### Describe the problem（描述问题）

在 ReportInfo 中获取 cache 会获取不到，或者会把 cache 删掉，是什么意思

### Midway Versions（Midway 版本）

✓ @midwayjs/faas-typings(not installed)
✓ @midwayjs/fc-starter(not installed)
✓ @midwayjs/serverless-http-parser(not installed)
✓ @midwayjs/async-hooks-context-manager(3.15.0)
✓ @midwayjs/axios(not installed)
✓ @midwayjs/bootstrap(3.15.0)
✓ @midwayjs/bull(not installed)
✓ @midwayjs/bull-board(not installed)
✖ @midwayjs/cache-manager(current: 3.15.6, allow: ["3.15.0","3.15.2","3.15.5"])
✖ @midwayjs/captcha(current: 3.15.6, allow: ["3.15.0","3.15.1","3.15.2","3.15.5"])
✓ @midwayjs/casbin(not installed)
✓ @midwayjs/casbin-redis-adapter(not installed)
✓ @midwayjs/casbin-typeorm-adapter(not installed)
✓ @midwayjs/code-dye(not installed)
✓ @midwayjs/consul(not installed)
✓ @midwayjs/core(3.15.0)
✓ @midwayjs/cos(not installed)
✓ @midwayjs/cron(not installed)
✓ @midwayjs/cross-domain(not installed)
✓ @midwayjs/decorator(3.15.0)
✓ @midwayjs/etcd(not installed)
✓ @midwayjs/express-session(not installed)
✓ @midwayjs/faas(not installed)
✓ @midwayjs/grpc(not installed)
✓ @midwayjs/http-proxy(not installed)
✖ @midwayjs/i18n(current: 3.15.7, allow: ["3.15.0","3.15.1","3.15.2"])
✓ @midwayjs/info(3.15.2)
✓ @midwayjs/jwt(3.15.2)
✓ @midwayjs/kafka(not installed)
✓ @midwayjs/leoric(not installed)
✓ @midwayjs/mikro(not installed)
✓ @midwayjs/mock(3.15.2)
✓ @midwayjs/mongoose(not installed)
✓ @midwayjs/mqtt(not installed)
✓ @midwayjs/oss(not installed)
✓ @midwayjs/otel(not installed)
✓ @midwayjs/passport(3.15.2)
✓ @midwayjs/process-agent(not installed)
✓ @midwayjs/prometheus(not installed)
✓ @midwayjs/prometheus-socket-io(not installed)
✓ @midwayjs/rabbitmq(not installed)
✓ @midwayjs/redis(not installed)
✓ @midwayjs/security(not installed)
✓ @midwayjs/sequelize(not installed)
✓ @midwayjs/session(3.15.2)
✓ @midwayjs/socketio(not installed)
✓ @midwayjs/static-file(not installed)
✓ @midwayjs/swagger(3.15.3)
✓ @midwayjs/tablestore(not installed)
✓ @midwayjs/tags(not installed)
✓ @midwayjs/typegoose(not installed)
✖ @midwayjs/typeorm(current: 3.15.6, allow: ["3.15.0","3.15.2"])
✓ @midwayjs/upload(not installed)
✓ @midwayjs/validate(3.15.2)
✓ @midwayjs/version(not installed)
✓ @midwayjs/view(not installed)
✓ @midwayjs/view-ejs(not installed)
✓ @midwayjs/view-nunjucks(not installed)
✓ @midwayjs/web(not installed)
✓ @midwayjs/express(not installed)
✓ @midwayjs/koa(3.15.2)
✓ @midwayjs/ws(not installed)

**Comment by czy88840616** [comment]
? 什么ReportInfo

**Comment by AgileZM** [comment]
拦截器呀，官网上写有的



---- 回复的原邮件 ----
| 发件人 | Harry ***@***.***> |
| 日期 | 2024年04月08日 22:43 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [midwayjs/midway] cache 失效 (Discussion #3755) |

? 什么ReportInfo
